src/components/NamePicker.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import TextField from '@material-ui/core/TextField';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+
+export default function NamePicker(): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="md">
+      <div className={classes.paper}>
+        <Typography component="h1" variant="h5">
+          Search our society's records
```

Comment:
This doesn't seem like it belongs to the Name Picker. It's a general comment that seems to belong to the entire form itself, not the name.

Reply:
I see it, will fix!